config/routes.rb

```diff
@@ -1,6 +1,16 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'users/groups'
+  get 'users/operations'
+  devise_for :users
+  resources :users, only: [ :index, :show] do
+    resources :groups, only: [ :index, :show, :new, :create] do
+      resources :operations, only: [ :index, :show, :new, :create]
+    end
+  end
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  namespace :users do
+    root :to => "home#index"
+  end
+  
+  root to: "home#index"
```

Comment:
- [ ] it would be better if you add a page with your name app and links to sign up and sign in in order to match the design and also the requirements 
- Screenshot 
<img width="1042" alt="Screen Shot 2022-10-13 at 16 16 21" src="https://user-images.githubusercontent.com/61790587/195637461-5b884897-e67e-407e-bf30-daa6b81a5d78.png">

Reply:
- Addressed 👍🏻